slack_sdk/oauth/installation_store/async_cacheable_installation_store.py

```diff
@@ -10,6 +10,7 @@
 class AsyncCacheableInstallationStore(AsyncInstallationStore):
```

Comment:
FYI: This is a decorator-pattern class for InstallationStore. It provides a simple cache layer to reduce real storage access.